crates/runtime-ffi/src/api.rs

```diff
@@ -555,8 +552,16 @@ pub unsafe extern "C" fn svm_create_account(
     catch_unwind_or_fail(|| {
         let runtime = RUNTIME_TRACKER.get(runtime_ptr).unwrap();
         let account_addr = Address::new(slice::from_raw_parts(addr, Address::N));
+        let template_addr = TemplateAddr::god_template();
         let counter = ((counter_upper_bits as u128) << 64) | (counter_lower_bits as u128);
-        runtime.create_account(&account_addr, "".to_string(), balance, counter)?;
+
+        runtime.create_account(
```

Comment:
`Runtime::create_account` implies that the template address is `TemplateAddr::god_template()` (or rather, whatever we'll end up using for Simple Coin Transfer). Passing is as an argument gives the impression that it's customizable, but it's not. What do you think about this?